cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: l7-lb-controller
+  namespace: kube-system
+  labels:
+    k8s-app: glbc
+    version: v0.5
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "GLBC"
+spec:
+  # There should never be more than 1 controller alive simultaneously.
+  replicas: 1
+  selector:
+    k8s-app: glbc
+    version: v0.5
+  template:
+    metadata:
+      labels:
+        k8s-app: glbc
+        version: v0.5
+        name: glbc
```

Comment:
my concern is that users will run `get pods` and see `glbc` and have no idea what it is.